x-pack/test/api_integration/apis/cases/privileges.ts

```diff
@@ -272,6 +371,86 @@ export default ({ getService }: FtrProviderContext): void => {
       });
     }
 
+    for (const { user, owner, userWithFullPerms } of [
+      { user: casesV3NoAssigneeUser, owner: CASES_APP_ID, userWithFullPerms: casesV3AllUser },
+      {
+        user: casesV2NoCreateCommentWithReopenUser,
+        owner: CASES_APP_ID,
+        userWithFullPerms: casesV3AllUser,
+      },
+      {
+        user: obsCasesV2NoCreateCommentWithReopenUser,
+        owner: OBSERVABILITY_APP_ID,
+        userWithFullPerms: obsCasesV3AllUser,
+      },
+      {
+        user: secCasesV2NoCreateCommentWithReopenUser,
+        owner: SECURITY_SOLUTION_APP_ID,
+        userWithFullPerms: secCasesV3AllUser,
+      },
+      { user: secReadUser, owner: SECURITY_SOLUTION_APP_ID, userWithFullPerms: secAllUser },
+      { user: casesOnlyDeleteUser, owner: CASES_APP_ID, userWithFullPerms: casesAllUser },
+      {
+        user: obsCasesOnlyDeleteUser,
+        owner: OBSERVABILITY_APP_ID,
+        userWithFullPerms: obsCasesAllUser,
+      },
+    ]) {
+      it(`User ${
+        user.username
+      } with role(s) ${user.roles.join()} CANNOT change assignee`, async () => {
+        const caseInfo = await createCase(supertest, getPostCaseRequest({ owner }));
+        const [{ uid: assigneeId }] = await suggestUserProfiles({
+          supertest: supertestWithoutAuth,
+          req: { name: userWithFullPerms.username, owners: [owner], size: 1 },
+          auth: { user: userWithFullPerms, space: null },
+        });
+        await updateCaseAssignee({
+          supertest: supertestWithoutAuth,
+          caseId: caseInfo.id,
+          assigneeId,
+          expectedHttpCode: 403,
+          auth: { user, space: null },
+          version: caseInfo.version,
+        });
+      });
+    }
+
+    for (const { user, owner } of [
+      { user: casesV3ReadAndAssignUser, owner: CASES_APP_ID },
+      { user: secAllUser, owner: SECURITY_SOLUTION_APP_ID },
+      { user: casesAllUser, owner: CASES_APP_ID },
+      { user: casesV2AllUser, owner: CASES_APP_ID },
+      { user: secCasesV2AllUser, owner: SECURITY_SOLUTION_APP_ID },
+      { user: obsCasesV2AllUser, owner: OBSERVABILITY_APP_ID },
+      { user: obsCasesAllUser, owner: OBSERVABILITY_APP_ID },
+      { user: secCasesV2AllUser, owner: SECURITY_SOLUTION_APP_ID },
+      { user: obsCasesAllUser, owner: OBSERVABILITY_APP_ID },
+      { user: obsCasesV2AllUser, owner: OBSERVABILITY_APP_ID },
+      { user: secCasesV3AllUser, owner: SECURITY_SOLUTION_APP_ID },
+      { user: casesV3AllUser, owner: CASES_APP_ID },
+      { user: obsCasesV3AllUser, owner: OBSERVABILITY_APP_ID },
```

Comment:
Nit: I think V1 and V2 variants of users: casesNoDelete, obsCasesNoDelete, and secAllCasesNoDelete (the names are sort of misleading, the have the minimal_all privilege) should be added here.

Reply:
that I agree with, I think I was just trying to avoid a permutation explosion in these test names, this is quickly going to become untenable how it currently works, i.e. secCasesV3MinAllWithSettingsWithoutCommentWithReopenWithoutDeleteWithAssign ha